src/pages/Library.js

```diff
@@ -5,26 +5,81 @@ import CollectionGrid from "../components/CollectionGrid";
 import CollectionItem from "../components/CollectionItem";
 import OptionBox from "../components/OptionBox";
 import SortModal from "../components/SortModal";
-
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+import BrowseEmpty from "../components/BrowseEmpty";
+import gameCollection from "../api/CollectionData";
+import gameWishlist from "../api/WishlistData";
+import browseData from "../api/FakeData";
 export default function Library() {
-  const [navigation, setNavigation] = useState("Collection");
+  const [navigation, setNavigation] = useState("");
   const [options, setOptions] = useState(false);
   const [showSort, setShowSort] = useState(false);
-  const GamesArray = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16];
+  const [visibility, setVisibility] = useState(false);
+  const [textInput, setTextInput] = React.useState("");
+
+  const searchContent = browseData.filter(info =>
+    info.name.toLowerCase().includes(textInput.toLowerCase())
+  );
+  let empty = textInput;
+  console.log(empty);
+  console.log(textInput);
```

Comment:
PRs should not include "console.logs" as long as they are not necessary for the product. Moreover, one of these chould be redundant, as in Line 23 empty is assigned to textInput.